packages/server/graphql/types/AcceptTeamInvitationPayload.ts

```diff
@@ -42,6 +42,9 @@ const AcceptTeamInvitationPayload = new GraphQLObjectType<any, GQLContext>({
         _args: unknown,
         {dataLoader, authToken}: GQLContext
       ) => {
+        if (!meetingId) {
+          return null
+        }
         const viewerId = getUserId(authToken)
         const meeting = await dataLoader.get('newMeetings').load(meetingId)
         if (!meeting) {
```

Comment:
I think we should remove the error message on line 48 because it's not an error if there's no meetingId

Reply:
I disagree, it's an error if we pass an invalid `meetingId`

Reply:
`meetingId` is coming from a [TeamInvitation](https://github.com/ParabolInc/parabol/blob/9177909d9b3bc9cf2ba5d88a32c0ce53b1da177f/packages/server/database/types/TeamInvitation.ts) where `meetingId` can be undefined

Reply:
Yes. We check for null `meetingId` in l45 and return if null was passed. If a supposedly valid `meetingId` is passed but we cannot find a meeting associated with it, that seems to be an error to me.

Reply:
Calling `loadNonNull` would do the trick as well.

Reply:
Yes, I think this is an error about meeting expected but it is not found for some reason.